docs/features/software-catalog/installation.md

```diff
@@ -217,8 +170,9 @@ Finally, start up the backend with the new configuration:
 
 ```bash
 cd packages/backend
+# or `yarn dev` to run both frontend and backend
```

Comment:
`yarn dev` should be run from the root, I think - so should be above the `cd packages/backend` IMO.